doc/source/rllib/rllib-algorithms.rst

```diff
@@ -217,13 +217,13 @@ Asynchronous Proximal Policy Optimization (APPO)
     **APPO architecture:** APPO is an asynchronous variant of :ref:`Proximal Policy Optimization (PPO) <ppo>` based on the IMPALA architecture,
     but using a surrogate policy loss with clipping, allowing for multiple SGD passes per collected train batch.
     In a training iteration, APPO requests samples from all EnvRunners asynchronously and the collected episode
```

Comment:
```suggestion
    In a training iteration, APPO requests samples from all EnvRunners asynchronously and RLlib returns the collected episode
```

Reply:
I feel like it's super obvious that the `EnvRunners` return the requested samples, b/c of the "APPO requests samples from **all EnvRunners** ... ".

Mentioning that "RLlib" does the returning is way too general. Everything is run by RLlib, so this doesn't give the user any useful information as to which particular component does the returning.

Reply:
"RLlib" was my guess and the point was just to clarify who's doing the returning, if needed. If it's obvious to the reader, just ignore my suggestion.